DigitalLearningSolutions.Web/ViewModels/Support/Resources/ResourcesCategoryViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Support.Resources
+{
+    using DigitalLearningSolutions.Data.Models.Support;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class ResourcesCategoryViewModel
+    {
+        public ResourcesCategoryViewModel(ResourceCategory resourceCategory, string currentSystemBaseUrl)

```

Comment:
Why not do the standard thing here, take in a category and IEnumerable<ResourceItemViewModel>?
Then in ResourcesService you can convert the list of resources directly to a list of ResourcesCategoryViewModel rather than going through `ResourceCategory`.

Reply:
Belay, VM not accessible from services.